### PR TITLE
Nyah voting

### PR DESCRIPTION
**Pull request by Basialo**
I made a draft version of the GameContext file with some useful variables, haven't done the panel yet though.

(is this what I'm supposed to do??)

### CODE REVIEW

File: client/src/components/DayVotingMenu.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, {useContext} from 'react'
+import Timer from './Timer'
+import VotePanel from './VotePanel'
+import Graveyard from './Graveyard'
+import '../style/DayVotingMenuStyle.css';
+import { GameContext } from '../GameContext'
+
+import { useState, useEffect } from 'react'
```
Comment (munjoonteo) [inline client/src/components/DayVotingMenu.js]: Combine imports

File: client/src/components/DayVotingMenu.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const DayVotingMenu = () => {
+    const { players, setPlayers } = useContext(GameContext)
```
Comment (munjoonteo) [inline client/src/components/DayVotingMenu.js]: Combine these three lines

File: client/src/components/DayVotingMenu.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            <div className="actionPanel">
+                <VotePanel/>
+                <Graveyard/>
+            </div>
+            <button className="confirm-button">Confirm</button>
+        </div>
+    )
+}
+
+export default DayVotingMenu
```
Comment (munjoonteo) [inline client/src/components/DayVotingMenu.js]: New line at end of file

File: client/src/components/Graveyard.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            <div className="graveyardScroll">
+                {tempDeadPlayers.map((pName) => (<div className="graveyardListItem">
+                        <h3>{pName}</h3>
+                </div>))}
+            </div>
+        </div>
+    )
+}
+
+export default Graveyard
```
Comment (munjoonteo) [inline client/src/components/Graveyard.js]: Here too

File: client/src/components/Timer.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+const Timer = ( {} ) => {
+    
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+export default Timer
```
Comment (munjoonteo) [inline client/src/components/Timer.js]: Here too

File: client/src/components/VotePanel.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+                {tempPlayerNames.map((pName) => (<div className="voteListItem">
+                    <div className="label"><h3>{pName}</h3></div>
+                    <input type="radio" id={pName} name="votee" value={pName}></input>
+                </div>))}
+            </div>
+        </div>
+    )
+}
+
+export default VotePanel
```
Comment (munjoonteo) [inline client/src/components/VotePanel.js]: Here too

File: client/src/components/DayVotingMenu.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import React, {useContext} from 'react'
+import Timer from './Timer'
+import VotePanel from './VotePanel'
+import Graveyard from './Graveyard'
+import '../style/DayVotingMenuStyle.css';
```
Comment (munjoonteo) [inline client/src/components/DayVotingMenu.js:5]: Where are these style files o.o

**Review by munjoonteo — CHANGES_REQUESTED** [review]

File: client/src/components/DayVotingMenu.js (hunk 7 above)
Comment (Basialo) [reply client/src/components/DayVotingMenu.js:5]: oops o.o

**Review by munjoonteo — APPROVED** [review]

**Review by munjoonteo — APPROVED** [review]

**Pull request merged**